apps/api/v2/src/ee/event-types/event-types_2024_06_14/controllers/event-types.controller.e2e-spec.ts

```diff
@@ -232,7 +248,11 @@ describe("Event types Endpoints", () => {
           expect(createdEventType.bookingFields).toEqual(body.bookingFields);
           expect(createdEventType.ownerId).toEqual(user.id);
           expect(createdEventType.scheduleId).toEqual(firstSchedule.id);
-
+          expect(createdEventType.bookingLimits).toEqual(body.bookingLimits);
+          expect(createdEventType.onlyShowFirstAvailableSlot).toEqual(body.onlyShowFirstAvailableSlot);
+          expect(createdEventType.durationLimits).toEqual(body.durationLimits);
+          expect(createdEventType.offsetStart).toEqual(body.offsetStart);
+          expect(createdEventType.bookingWindow).toEqual(body.bookingWindow);
```

Comment:
can we have similar test for update and get endpoint

Reply:
added 👍